app/src/main/java/com/hedvig/app/ui/compose/designsystem/LargeOutlinedButton.kt

```diff
@@ -0,0 +1,46 @@
+package com.hedvig.app.ui.compose.designsystem
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.HedvigTheme
+
+@Composable
+fun LargeOutlinedButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    content: @Composable RowScope.() -> Unit,
+) {
+    OutlinedButton(
+        onClick = onClick,
+        modifier = modifier,
```

Comment:
What do you think about putting `.fillMaxWidth()` on this modifier? I think every use-case for the `Large Outlined Button` is actually `fillMaxWidth()`, this might save us some effort on the call-sites 🤔